config/locales/es.yml

```
@@ -1624,6 +1625,7 @@
         weaknesses: "Observaciones"
         oportunities: "Oportunidades de mejora"
         work_papers: "Papeles de trabajo"
+        scored_business_unit_id: 'Unidad de negocio evaluada'
```

```suggestion
        scored_business_unit_id: "Unidad de negocio evaluada"
```
Solo por consistencia con el resto.

---

perfecto, me lo comi